draft-ietf-ice-rfc5245bis.xml

```diff
@@ -999,53 +984,23 @@ following exceptions:
 <section title="Server Reflexive and Relayed Candidates">
 
 <t>
-Agents SHOULD obtain relayed candidates and SHOULD obtain server
-reflexive candidates. These requirements are at SHOULD strength to
-allow for provider variation. Use of STUN and TURN servers may be
-unnecessary in closed networks where agents are never connected to the
-public Internet or to endpoints outside of the closed network. In such
-cases, a full implementation would be used for agents that are
-dual-stack or multihomed, to select a host candidate. Use of TURN
-servers is expensive, and when ICE is being used, they will only be
-utilized when both endpoints are behind NATs that perform address and
-port dependent mapping. Consequently, some deployments might consider
-this use case to be marginal, and elect not to use TURN servers. If an
-agent does not gather server reflexive or relayed candidates, it is
-RECOMMENDED that the functionality be implemented and just disabled
-through configuration, so that it can be re-enabled through
-configuration if conditions change in the future.
+An agent SHOULD gather server reflexive and relayed candidates.
 </t>
 
 <t>
-If an agent is gathering both relayed and server reflexive candidates,
-it uses a TURN server. If it is gathering just server reflexive
-candidates, it uses a STUN server.
-</t>
-
-<t>
-The agent next pairs each host candidate with the STUN or TURN server
-with which it is configured or has discovered by some means. If a STUN
-or TURN server is configured, it is RECOMMENDED that a domain name be
+The agent pairs each host candidate with the STUN or TURN servers
+with which it is configured or has discovered by some means. 
+It is RECOMMENDED that a domain name be
 configured, and the DNS procedures in <xref target="RFC5389"/> (using
 SRV records with the "stun" service) be used to discover the STUN
 server, and the DNS procedures in <xref target="RFC5766"/> (using SRV
 records with the "turn" service) be used to discover the TURN server.
 </t>
 
-<t>This specification only considers usage of a single STUN or TURN
-server. When there are multiple choices for that single STUN or TURN
```

Comment:
clearly we have to change the first sentence that one can only have a single server